1/Main.java

```diff
@@ -0,0 +1,36 @@
+package first;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+
+public class Main {
+    public static void main(String[] args) {
+        while (true) {
+            try (BufferedReader reader = new BufferedReader(new InputStreamReader(System.in))) {
+                String binaryNumber = reader.readLine();
+                if(!isValidBinaryNumber(binaryNumber)) System.out.println("Error, not valid binary number");
```

Comment:
format files , add parentheses {}

Reply:
in all tasks

Reply:
Thank you very much. 

I formated files by adding parentheses or using the ternary operator.

Something like this
``` 
return (target == 1) ? num | (1 << pos) : 
                       num & (~(1 << pos));
```